ChangeFeedLabSolution/ChangeFeedFunction.csproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>

```

Comment:
If there a reason why we would use a down-level framework?   I'd recommend bumping to `netcoreapp3.1` and targeting Functions v3 for the latest LTS.  If we're set on targeting Functions 2, then we should be using `netcoreapp2.1`.    We shouldn't be targeting Functions v1 in a sample.   
_([Azure Functions supported versions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v2%2Ccmd#supported-versions))_